### PR TITLE
#15-feat/user-endpoints

### PR DESCRIPTION
**Pull request by NicolasFontana**

### CODE REVIEW

**Comment by tomasbirbe** [comment]
delete sensible data from dbconfig.ts

File: server/src/controllers/userController.ts [hunk 1]
```diff
@@ -0,0 +1,153 @@
+import { Request, Response } from "express";
+import { getUserService, createUserService, deleteUserService, updateUserService } from "../services/userService";
+import { User } from "../models/userModel";
+
+const getUsers = async (req: Request, res: Response) => {
+    try {
+        const { id } = req.params;
+        const user = await getUserService(id);
+        if (!id) {
```
Comment (tomasbirbe) [inline server/src/controllers/userController.ts:9]: Up this code one line

File: server/src/routes/users.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import express from "express";
+import userController from "../controllers/userController";
+
+const router = express.Router();
+
+router
+  .get("/:id?", userController.getUsers)
```
Comment (tomasbirbe) [inline server/src/routes/users.ts]: Add route for GET / to get one user and remove extra logic on userController.getUsers

File: server/src/controllers/userController.ts [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    try {
+        const users = await getUsersService();
+        if (users.length > 0) {
+            return res.status(200).json({
+                message: "Users found",
+                data: users,
+                error: false,
+            });
+        }
+        return res.status(200).json({
```
Comment (AdrianFerrari) [inline server/src/controllers/userController.ts:47]: esto tiene que mandar 404

**Review by AdrianFerrari — APPROVED** [review]
&#9989; aproved

**Pull request merged**